creator/utils.py

```diff
@@ -0,0 +1,85 @@
+import time
+import logging
+from rq.job import NoSuchJobError, JobStatus
+from rq.command import send_kill_horse_command
+from rq.worker import Worker, WorkerStatus
+
+WAIT_FOR_FAIL_TIMEOUT = 30
+
+
+def stop_job(job_id, queue=None, delete=True, logger=None):
```

Comment:
What is the use case for passing a logger? This may cause unexpected behavior when trying to modify our logging setup.

Reply:
I'll remove it